listenbrainz_spark/stats/user/entity.py

```diff
@@ -102,9 +101,39 @@ def get_entity_all_time(entity):
 
     handler = entity_handler_map[entity]
     data = handler(table_name)
-    messages = create_messages(data=data, entity=entity, stats_type='user_entity', stats_range='all_time',
+    messages = create_messages(data=data, entity=entity, stats_range='all_time',
                                from_ts=from_date.timestamp(), to_ts=to_date.timestamp())
 
     current_app.logger.debug("Done!")
 
     return messages
+
+
+def create_messages(data, entity, stats_range, from_ts, to_ts):
```

Comment:
Shall we mention the argument types here?

Reply:
I am planning to do a follow up PR for that. Didn't wan't make this PR big.